.github/workflows/ci.yml

```diff
@@ -17,6 +17,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
```

Comment:
ça doit accélérer pas pire ça ?

Reply:
> It must speed things up, does it?
The repos are still fairly small with no image resources so it doesn't make a lot of difference but it's a good thing for the future.